onegov/people/models/agency.py

```diff
@@ -77,19 +78,37 @@ def organigram_file(self, value):
     def add_person(self, person_id, title, **kwargs):
         """ Appends a person to the agency with the given title. """
 
-        order = kwargs.pop('order', 2 ** 16)
+        order_within_agency = kwargs.pop('order_within_agency', 2 ** 16)
+        session = object_session(self)
+
+        orders_for_person = session.query(
+            AgencyMembership.order_within_person
+        ).filter_by(person_id=person_id).all()
```

Comment:
You don't need to use `.all()`, which just turns the query into a list. If you remove it your code would work the exact same way, but use less memory.